docs/DeveloperGuide.md

```diff
@@ -627,29 +722,27 @@ The predicate is then used to filter the list of person.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## **Appendix: Requirements**
+## **Appendix A: Requirements**
 
 ### Product scope
 
 **Target user profile**:
 
-A private 1-to-1 home tuition teacher in Singapore that:
+A private 1-to-1 home tuition teacher that:
 
 1. has a need to manage not more than 50 student contacts
-2. has a need to manage admin details of each student, namely, lesson fees and payment
+2. has a need to manage payment details of each student
```

Comment:
isn't the original clearer?

Reply:
I think fees and payment about the same so it's fine